apps/crn-server/test/controllers/research-outputs.test.ts

```diff
@@ -711,6 +716,48 @@ describe('ResearchOutputs controller', () => {
         researchOutputs.create(researchOutputRequest),
       ).rejects.toThrow('Internal Server');
     });
+    test.each<{ identifierType: ResearchOutputIdentifierType; field: string }>([
+      { identifierType: ResearchOutputIdentifierType.DOI, field: 'doi' },
+      { identifierType: ResearchOutputIdentifierType.RRID, field: 'rrid' },
+      {
+        identifierType: ResearchOutputIdentifierType.LabCatalogNumber,
+        field: 'labCatalogNumber',
+      },
+      {
+        identifierType: ResearchOutputIdentifierType.AcessionNumber,
+        field: 'accession',
+      },
+    ])('should set the field properly', async ({ identifierType, field }) => {
```

Comment:
maybe something like `should set the ${field} when the ${identifierType} type is selected`?

also I find this test hard to read - that's only cause of the .each with type declaration and an array definition typed inline, maybe extract them above?